internal/pkg_npm/pkg_npm.bzl

```diff
@@ -123,6 +123,12 @@ PKG_NPM_ATTRS = {
         doc = """Files inside this directory which are simply copied into the package.""",
         allow_files = True,
     ),
+    "hide_build_files": attr.bool(
+        doc = """If set BUILD and BUILD.bazel files are prefixed with `_` in the npm package.
+        The default is True since npm packages that contain BUILD files don't work with
+        `yarn_install` and `npm_install` without a post-install step that deletes or renames them.""",
```

Comment:
opt: reference the new feature landed in Bazel (coming in 2.1?) which makes this obsolete?

Reply:
👍

Reply:
It's already merged: https://github.com/bazelbuild/bazel/pull/10261
I updated your comment here